cli/internal/run/run.go

```diff
@@ -426,22 +437,25 @@ or non-zero exit code. The default behavior is to bail`
 	_dryRunHelp = `List the packages in scope and the tasks that would be run,
 but don't actually run them. Passing --dry=json or
 --dry-run=json will render the output in JSON format.`
+	_graphHelp       = `Generate a graph of the task execution and output to a file when specified, otherwise stdout`
```

Comment:
I think we ought to include some information about the kinds of filenames that are supported. `.jpg`, `.html`, `.svg`, and a note that we're delegating to `dot` if they want more information.